forge-gui/res/languages/en-US.properties

```diff
@@ -2965,4 +2965,6 @@ lblEmblem=Emblem
 lblBoon=Boon
 lblExitToWoldMap=Exit to the World Map?
 lblWouldYouLikeDestroy=Would you like to destroy {0}?
-lblAdventureDescription=Adventure mode is where you explore the ever-changing landscape of Shandalar, and duel creatures to gain gold, shards, equipment and new cards to become the best and collect them all!
\ No newline at end of file
+lblAdventureDescription=Adventure mode is where you explore the ever-changing landscape of Shandalar, and duel creatures to gain gold, shards, equipment and new cards to become the best and collect them all!
+lblDownloadMusic=Download the custom Forge soundtrack.
+btnDownloadMusic=Download Custom Forge Soundtrack.
```

Comment:
Cstom music and sounds resides in the cache, they need different paths and own labels since forge custom music sets and sounds are selectable, adventure has a single BGM set and there's no customization implemented (unless the user overwrite the actual music files or create a specific cache folder for adventure custom sounds)

Reply:
Thanks for the feedback. I'll just limit this PR to match and battle tracks for both Desktop and Mobile and will later try to get it wired up for adventure (mobile exclusive)